public/css/primary.css

```diff
@@ -1,44 +0,0 @@
-header {
```

Comment:
I was able to edit secondary.css to include a `landing` class that changes the behavior, so this could be deduplicated from secondary.css.